packages/inventory/src/Stocktake/api/api.ts

```diff
@@ -170,6 +170,17 @@ export const getStocktakeQueries = (sdk: Sdk, storeId: string) => ({
     });
     const { insertStocktake } = result;
     if (insertStocktake.__typename === 'StocktakeNode') {
+      if (itemIds) {
+        const insertStocktakeLines = itemIds.map(itemId => ({
+          id: FnUtils.generateUUID(),
+          stocktakeId: insertStocktake.id,
+          itemId,
+        }));
+        await sdk.upsertStocktakeLines({
```

Comment:
the return shape makes error catching difficult here, so I didn't bother 😁 
The mutation always returns a BatchMutationResponse regardless of errors, and you have to drill down to get the insert lines response. Decided that it doesn't really matter anyway - the key mutation here is the stocktake insert, and if the lines insert fails that's pretty obvious.